javascript/javascript2/week3/fastestpresser.js

```diff
@@ -0,0 +1,78 @@
+const input = document.getElementById("input-textbox");
+const button = document.getElementById("btn");
+const countS = document.getElementById("counter-s");
+const countL = document.getElementById("counter-l");
+const messageForS = document.getElementById("winning-message-s");
+const messageForL = document.getElementById("winning-message-l");
+let confettiGenerator;
+let counterS = 0;
+let counterL = 0;
+let gameEnd = false;
+let gameStart = false;
+
```

Comment:
I really liked this way of programming of having different states of the game. It is usually a very good way of thinking to solve the problems like this. 😄